src/main/java/com/csc366/project/entity/LineItem.java

```diff
@@ -0,0 +1,136 @@
+package com.csc366.project.entity;
+
+import com.csc366.project.entity.key.LineItemKey;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.StringJoiner;
+import java.util.Date;
+import java.util.Objects;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.persistence.CascadeType;
+import javax.persistence.FetchType;
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.ManyToOne;
+import javax.persistence.JoinColumn;
+import javax.persistence.MapsId;
+import javax.persistence.EmbeddedId;
+
+@Entity
+@Table(name = "lineItem")
+public class LineItem{
+
+    
+    @EmbeddedId
+    private LineItemKey key;
+
+    @ManyToOne
+    @MapsId("sku")
+    @JoinColumn(name = "sku")
+    private Product product;
```

Comment:
We should add a `@OneToMany` lineItem property on the `Product` class if we're doing this I think

Reply:
agreed just added it